portfolio/src/main/webapp/script.js

```diff
@@ -26,21 +26,21 @@ function getRandomQuote() {
     // see function doc on line 57
     revertToUnchecked();
 
-    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    let quote = quotes[Math.floor(Math.random() * quotes.length)];
   
-    let entrance = entrances[Math.floor(Math.random() * greetings.length)];
+    let entrance = entrances[Math.floor(Math.random() * entrances.length)];
 
     
-    let quote = document.getElementById('quote');
+    let quoteDiv = document.getElementById('quote');
 
     // I first clear the quote div's classList in order to assign a random/new css animation from the entrances array. 
```

Comment:
Describe the code objectively. For e.g., // Clears the quote div's ...

Reply:
changed description